.travis.yml

```diff
@@ -10,6 +10,7 @@ cache:
 before_install:
 - unset GEM_PATH GEM_HOME IRBRC JRUBY_OPTS
 - rm ~/.m2/settings.xml
+- rvm install 2.1.0
```

Comment:
2.1 is easily available in our internal CI system, so I picked that so we could use the same version on both. Are there any practical problems right now with running jt (the only code we use it for) on this version?